syncall/scripts/tw_caldav_sync.py

```diff
@@ -192,6 +196,10 @@ def main(
     caldav_passwd = os.environ.get("CALDAV_PASSWD")
     if caldav_passwd is not None:
         logger.debug("Reading the caldav password from environment variable...")
+    elif caldav_passwd_cmd is not None:
+        caldav_passwd = subprocess.check_output(caldav_passwd_cmd, shell=True, text=True)
```

Comment:
Instead of raising an uncaught exception, could we instead use `subprocess.run`, check whether the command succeeded or not and if it didn't use `error_and_exit` to inform the user about this failure ?

Reply:
Sounds good. Done.